stable/database/README.md

```diff
@@ -270,6 +270,9 @@ The following tables list the configurable parameters of the `database` chart an
 | `sm.tolerations` | Tolerations for NuoDB SM | `[]` |
 | `sm.topologySpreadConstraints` | Topology spread constraints for NuoDB SM | `[]` |
 | `sm.readinessTimeoutSeconds` | SM readiness probe timeout, sometimes needs adjusting depending on environment and pod resources | `5` |
+| `sm.storageGroup.enabled` | Enable Table Partitions and Storage Groups (TPSG) for all SMs in this database Helm release | `false` |
+| `sm.storageGroup.name` | The name of the storage group. Only alphanumeric and underscore ('_') characters are allowed. By default the Helm release name is used | `.Release.Name` |
+| `te.enablePod` | Create deployment for TEs | `true` |
```

Comment:
`enablePod` sounds like a weird name for the value, since this is actually enabling/disabling the TE deployment. Creation of TE pods can also be inhibited by setting `te.replicas=0`, so `enableDeployment` sounds more descriptive.

I guess this is consistent with `enablePod` which already exists for SMs. Also, I noticed that the `sm.*.enablePod` rows both mention DaemonSets, which we no longer support.

Reply:
> I guess this is consistent with enablePod which already exists for SMs

Right. I personally prefer `enabled` but decided to stay consistent.

> Also, I noticed that the sm.*.enablePod rows both mention DaemonSets (DS), which we no longer support

Thanks for pointing this out. I will fix it.